docs/about/release-notes.md

```diff
@@ -107,6 +107,8 @@ to support such customization.
 * Bugfix: Support SOURCE_DATE_EPOCH environment variable for "reproducible"
   builds (#938)
 * Follow links when copying media files (#869).
+* Change "Edit on..." links to point directly to the file in the source
+  repository, rather than to the root of the repository (#975).
```

Comment:
This is fine. However, it might be good to point out that a new config setting has been added to make this possible. Maybe add:

    ... configurable via the new [`edit_uri`](../user-guide/configuration.md#edit_uri) setting.